components/compositing/compositor.rs

```diff
@@ -566,6 +566,19 @@ impl IOCompositor {
                 self.global.borrow_mut().send_transaction(txn);
             },
 
+            CompositorMsg::UpdateEpoch {
```

Comment:
Why do we need new IPC msg? Can we not pack this into SendDisplayList and/or GenerateFrame?